src/polus/plugins/_plugins/classes/plugin_methods.py

```diff
@@ -164,29 +194,103 @@ def __setattr__(self, name, value):
         if name == "_fs":
             if not issubclass(type(value), fsspec.spec.AbstractFileSystem):
                 raise ValueError("_fs must be an fsspec FileSystem")
-            else:
-                for i in self.inputs:
-                    i._fs = value
-                for o in self.outputs:
-                    o._fs = value
-                return
+            for i in self.inputs:
+                i._fs = value
+            for o in self.outputs:
+                o._fs = value
+            return
 
-        elif name != "_io_keys" and hasattr(self, "_io_keys"):
+        if name != "_io_keys" and hasattr(self, "_io_keys"):
             if name in self._io_keys:
                 logger.debug(
-                    "Value of %s in %s set to %s"
-                    % (name, self.__class__.__name__, value)
+                    f"Value of {name} in {self.__class__.__name__} set to {value}"
                 )
                 self._io_keys[name].value = value
                 return
-            else:
-                raise IOKeyError(
-                    "Attempting to set %s in %s but %s is not a valid I/O parameter"
-                    % (name, self.__class__.__name__, name)
-                )
+            raise IOKeyError(
+                f"Attempting to set {name} in {self.__class__.__name__} but"
+                "{name} is not a valid I/O parameter"
+            )
 
         super().__setattr__(name, value)
 
+    def _to_cwl(self):
+        """Return CWL yml as dict."""
```

Comment:
Ok I am not sure what is going on here. When pulling the PR locally, I don't see those files which seems to belong to another PR.